<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FloatEditWidget</class>
 <widget class="QWidget" name="FloatEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>29</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBox0">
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-99999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox1">
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-99999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QDoubleSpinBox" name="doubleSpinBox2">
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-99999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelSeparator0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true">,</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelSeparator1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true">,</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections/>
</ui>
